health/micro-ui-react19/web/packages/css/src/index.scss

```
@@ -18,3 +18,4 @@
 @import "./pages/employee/payment.scss";
 @import "./pages/employee/healthdss.scss";
 @import "./pages/employee/mycampaignsnew.scss";
+@import "./pages/employee/commodityManagement.scss";
```

Stylelint (17.7.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)